### DISCUSSION TITLE
Starting with empty fields on screen

### DISCUSSION THREAD

**Discussion by jeanbruder** [KRTirtho/spotube#1433] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Aplication is starting, but each field of the software if empty and seems waiting for some data (no picture, no text, no laylist).

### Expected Behavior

Working software

### Steps to reproduce

1. Open the app
2. Wait ...

### Operating System

Android

### Spotube version

3.5.0 release 29

### Installation source

F-Droid (Android)

### Additional information

============================== CATCHER 2 LOG ==============================
Crash occurred on 2024-03-23 13:50:48.700202

------- DEVICE INFO -------
id: SP1A.210812.016
board: merlinnfc
bootloader: unknown
brand: Redmi
device: merlinnfc
display: SP1A.210812.016
fingerprint: Redmi/merlinnfc_eea/merlinnfc:12/SP1A.210812.016/V13.0.3.0.SJOEUXM:user/release-keys
hardware: mt6768
host: non-pangu-pod-p7c98
isPhysicalDevice: true
manufacturer: Xiaomi
model: M2003J15SC
product: merlinnfc_eea
tags: release-keys
type: user
versionBaseOs:
versionCodename: REL
versionIncremental: V13.0.3.0.SJOEUXM
versionPreviewSdk: 0
versionRelease: 12
versionSdk: 31
versionSecurityPatch: 2022-11-01

------- APP INFO -------
environment: release
version: 3.5.0
appName: Spotube
buildNumber: 29
packageName: oss.krtirtho.spotube

---------- ERROR ----------
PathNotFoundException: Cannot retrieve length of file, path = 'imageUrl' (OS Error: No such file or directory, errno
------------ STACK TRACE -------
#0      _checkForErrorResponse (dart:io/common.dart:55)
#1      _File.length.<anonymous closure> (dart:io/file_impl.dart:389)
<asynchronous suspension>
#2      FileImage._loadAsync (package:flutter/src/painting/image_provider.dart:1484)
<asynchronous suspension>
#3      MultiFrameImageStreamCompleter._handleCodecReady (package:flutter/src/painting/image_stream.dart:985)
<asynchronous suspension>

------- CUSTOM INFO -------
======================================================================
============================== CATCHER 2 LOG ==============================
Crash occurred on 2024-03-24 14:28:20.284878

------- DEVICE INFO -------
id: SP1A.210812.016
board: merlinnfc
bootloader: unknown
brand: Redmi
device: merlinnfc
display: SP1A.210812.016
fingerprint: Redmi/merlinnfc_eea/merlinnfc:12/SP1A.210812.016/V13.0.3.0.SJOEUXM:user/release-keys
hardware: mt6768
host: non-pangu-pod-p7c98
isPhysicalDevice: true
manufacturer: Xiaomi
model: M2003J15SC
product: merlinnfc_eea
tags: release-keys
type: user
versionBaseOs:
versionCodename: REL
versionIncremental: V13.0.3.0.SJOEUXM
versionPreviewSdk: 0
versionRelease: 12
versionSdk: 31
versionSecurityPatch: 2022-11-01

------- APP INFO -------
environment: release
version: 3.5.0
appName: Spotube
buildNumber: 29
packageName: oss.krtirtho.spotube

---------- ERROR ----------
PathNotFoundException: Cannot retrieve length of file, path = 'url' (OS Error: No such file or directory, errno = 2)

------- STACK TRACE -------
#0      _checkForErrorResponse (dart:io/common.dart:55)
#1      _File.length.<anonymous closure> (dart:io/file_impl.dart:389)
<asynchronous suspension>
#2      FileImage._loadAsync (package:flutter/src/painting/image_provider.dart:1484)
<asynchronous suspension>
#3      MultiFrameImageStreamCompleter._handleCodecReady (package:flutter/src/painting/image_stream.dart:985)
<asynchronous suspension>

------- CUSTOM INFO -------
======================================================================
====
============================== CATCHER 2 LOG ==============================
Crash occurred on 2024-03-24 14:28:20.647639

------- DEVICE INFO -------
id: SP1A.210812.016
board: merlinnfc
bootloader: unknown
brand: Redmi
device: merlinnfc
display: SP1A.210812.016
fingerprint: Redmi/merlinnfc_eea/merlinnfc:12/SP1A.210812.016/V13.0.3.0.SJOEUXM:user/release-keys
hardware: mt6768
host: non-pangu-pod-p7c98
isPhysicalDevice: true
manufacturer: Xiaomi
model: M2003J15SC
product: merlinnfc_eea
tags: release-keys
type: user
versionBaseOs:
versionCodename: REL
versionIncremental: V13.0.3.0.SJOEUXM
versionPreviewSdk: 0
versionRelease: 12
versionSdk: 31
versionSecurityPatch: 2022-11-01

------- APP INFO -------
environment: release
version: 3.5.0
appName: Spotube
buildNumber: 29
packageName: oss.krtirtho.spotube

---------- ERROR ----------
FormatException: Unexpected character (at character 1)
Too many requests
^

------- STACK TRACE -------
#0      _ChunkedJsonParser.fail (dart:convert-patch/convert_patch.dart:1376)
#1      _ChunkedJsonParser.parseNumber (dart:convert-patch/convert_patch.dart:1243)
#2      _ChunkedJsonParser.parse (dart:convert-patch/convert_patch.dart:908)
#3      _parseJson (dart:convert-patch/convert_patch.dart:35)
#4      JsonDecoder.convert (dart:convert/json.dart:610)
#5      JsonCodec.decode (dart:convert/json.dart:216)
#6      SpotifyApiBase.handleResponseWithBody (package:spotify/src/spotify_base.dart:276)
#7      SpotifyApiBase._requestWrapper (package:spotify/src/spotify_base.dart:256)
<asynchronous suspension>
#8      SpotifyApiBase._getImpl (package:spotify/src/spotify_base.dart:210)
<asynchronous suspension>
#9      Pages.getPage (package:spotify/src/endpoints/endpoint_paging.dart:236)
<asynchronous suspension>
#10     SinglePages.stream.handlePageAndGetNext (package:spotify/src/endpoints/endpoint_paging.dart:171)
<asynchronous suspension>

------- CUSTOM INFO -------
======================================================================
============================== CATCHER 2 LOG ==============================
============================== CATCHER 2 LOG
------- DEVICE INFO -------
id: SP1A.210812.016
board: merlinnfc
bootloader: unknown
brand: Redmi
device: merlinnfc
display: SP1A.210812.016
fingerprint: Redmi/merlinnfc_eea/merlinnfc:12/SP1A.display: SP1A.210812.016
fingerprint: Redmi/merlinnfc_eea/merlinhost: non-pangu-pod-p7c98
isPhysicalDevice: true
manufacturer: Xiaomi
model: M2003J15SC
product: merlinnfc_eea
tags: release-keys
type: user
versionBaseOs:
versionCodename: REL
versionIncremental: V13.0.3.0.SJOEUXM
versionPreviewSdk: 0
versionRelease: 12
versionSdk: 31
versionSecurityPatch: 2022-11-01

------- APP INFO -------
environment: release
version: 3.5.0
appName: Spotube
buildNumber: 29
packageName: oss.krtirtho.spotube

---------- ERROR ----------
PathNotFoundException: Cannot retrieve length of---------- ERROR ----------
PathNotFoundException: Cannot retrieve len------- STACK TRACE -------
#0      _checkForErrorResponse (dart:io/common.dart:5------- STACK TRACE -------
#0      _checkForErrorResponse (dart:io/common.dart:55)
#1      _File.length.<anonymous closure> (dart:io/file_impl.dart:389)
<asynchronous suspension>
#2      FileImage._loadAsync (package:flutter/src/painting/image_provider.dart:1484)
<asynchronous suspension>
#3      MultiFrameImageStreamCompleter._handleCodecReady (package:flutter/src/painti=====================================================------- CUSTOM INFO -------
======================================================================
============================== CATCHER 2 LOG ==============================
Crash occurred on 2024-03-25 08:41:37.187596

------- DEVICE INFO -------
id: SP1A.210812.016
board: merlinnfc
bootloader: unknown
brand: Redmi
device: merlinnfc
display: SP1A.210812.016
fingerprint: Redmi/merlinnfc_eea/merlinnfc:12/SP1A.210812.016/V13.0.3.0.SJOEUXM:user/release-keys
hardware: mt6768
host: non-pangu-pod-p7c98
isPhysicalDevice: true
manufacturer: Xiaomi
model: M2003J15SC
product: merlinnfc_eea
tags: release-keys
type: user
versionBaseOs:
versionCodename: REL
versionIncremental: V13.0.3.0.SJOEUXM
versionPreviewSdk: 0
versionRelease: 12
versionSdk: 31
versionSecurityPatch: 2022-11-01

------- APP INFO -------
environment: release
version: 3.5.0
appName: Spotube
buildNumber: 29
packageName: oss.krtirtho.spotube

---------- ERROR ----------
FormatException: Unexpected character (at character 1)
Too many requests
^

------- STACK TRACE -------
#0      _ChunkedJsonParser.fail (dart:convert-patch/convert_patch.dart:1376)
#1      _ChunkedJsonParser.parseNumber (dart:convert-patch/convert_patch.dart:1243)
#2      _ChunkedJsonParser.parse (dart:convert-patch/convert_patch.dart:908)
#3      _parseJson (dart:convert-patch/convert_patch.dart:35)
#4      JsonDecoder.convert (dart:convert/json.dart:610)
#5      JsonCodec.decode (dart:convert/json.dart:216)
#6      SpotifyApiBase.handleResponseWithBody (package:spotify/src/spotify_base.dart:276)
#7      SpotifyApiBase._requestWrapper (package:spotify/src/spotify_base.dart:256)
<asynchronous suspension>
#8      SpotifyApiBase._getImpl (package:spotify/src/spotify_base.dart:210)
<asynchronous suspension>
#9      Pages.getPage (package:spotify/src/endpoints/endpoint_paging.dart:236)
<asynchronous suspension>
#10     AlbumQueries.newReleases.<anonymous closure> (package:spotube/services/queries/album.dart:94)
<asynchronous suspension>
#11     _AsyncCompleter.complete (dart:async/future_impl.dart:41)
<asynchronous suspension>
#12     Retryer.retryOperation (package:fl_query/src/core/mixins/retryer.dart:17)
<asynchronous suspension>
#13     CancelableCompleter.complete.<anonymous closure> (package:async/src/cancelable_operation.dart:425)
<asynchronous suspension>

------- CUSTOM INFO -------
======================================================================
============================== CATCHER 2 LOG ==============================
Crash occurred on 2024-03-25 10:38:57.562735

------- DEVICE INFO -------
id: SP1A.210812.016
board: merlinnfc
bootloader: unknown
brand: Redmi
device: merlinnfc
display: SP1A.210812.016
fingerprint: Redmi/merlinnfc_eea/merlinnfc:12/SP1A.210812.016/V13.0.3.0.SJOEUXM:user/release-keys
hardware: mt6768
host: non-pangu-pod-p7c98
isPhysicalDevice: true
manufacturer: Xiaomi
model: M2003J15SC
product: merlinnfc_eea
tags: release-keys
type: user
versionBaseOs:
versionCodename: REL
versionIncremental: V13.0.3.0.SJOEUXM
versionPreviewSdk: 0
versionRelease: 12
versionSdk: 31
versionSecurityPatch: 2022-11-01

------- APP INFO -------
environment: release
version: 3.5.0
appName: Spotube
buildNumber: 29
packageName: oss.krtirtho.spotube

---------- ERROR ----------
PathNotFoundException: Cannot retrieve length of file, path = 'imageUrl' (OS Error: No such file or directory, errno
------------ STACK TRACE ---#0   #0      _checkForErrorResponse (dart:io/common.dart:55)
#1      _File.length.<anonymous closure> (dart:io/file_impl.dart:389)
<asynchronous suspension>
#2      FileImage._loadAsync (package:flutter/src/painting/image_provider.dart:1484)
<asynchronous suspension>
#3      MultiFrameImageStreamCompleter._handleCodecReady (package:flutter/src/painting/image_stream.dart:985)
<asynchronous suspensi------------ CUSTOM INFO ---===========================================================================
============================== CATCHER 2 LOG ==============================
Crash occurred on 2024-03-25 10:38:58.122215

------- DEVICE INFO -------
id: SP1A.210812.016
board: merlinnfc
bootloader: unknown
brand: Redmi
device: merlinnfc
display: SP1A.210812.016
fingerprint: Redmi/merlinnfc_eea/merlinnfc:12/SP1A.210812.016/V13.0.3.0.SJOEUXM:user/release-keys
hardware: mt6768
host: non-pangu-pod-p7c98
isPhysicalDevice: true
manufacturer: Xiaomi
model: M2003J15SC
product: merlinnfc_eea
tags: release-keys
type: user
versionBaseOs:
versionCodename: REL
versionIncremental: V13.0.3.0.SJOEUXM
versionPreviewSdk: 0
versionRelease: 12
versionSdk: 31
versionSecurityPatch: 2022-11-01

------- APP INFO -------
environment: release
version: 3.5.0
appName: Spotube
buildNumber: 29
packageName: oss.krtirtho.spotube

---------- ERROR ----------
FormatException: Unexpected character (at character 1)
Too many requests
^

------- STACK TRACE -------
#0      _ChunkedJsonParser.fail (dart:convert-patch/convert_patch.dart:1376)
#1      _ChunkedJsonParser.parseNumber (dart:convert-patch/convert_patch.dart:1243)
#2      _ChunkedJsonParser.parse (dart:convert-patch/convert_patch.dart:908)
#3      _parseJson (dart:convert-patch/convert_patch.dart:35)
#4      JsonDecoder.convert (dart:convert/json.dart:610)
#5      JsonCodec.decode (dart:convert/json.dart:216)
#6      SpotifyApiBase.handleResponseWithBody (package:spotify/src/spotify_base.dart:276)
#7      SpotifyApiBase._requestWrapper (package:spotify/src/spotify_base.dart:256)
<asynchronous suspension>
#8      SpotifyApiBase._getImpl (package:spotify/src/spotify_base.dart:210)
<asynchronous suspension>
#9      Pages.getPage (package:spotify/src/endpoints/endpoint_paging.dart:236)
<asynchronous suspension>
#10     SinglePages.stream.handlePageAndGetNext (package:spotify/src/endpoints/endpoint_paging.dart:171)
<asynchronous suspension>

------- CUSTOM INFO -------
======================================================================

### Self grab

- [X] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Try login with Spotify account, unfortunately due to recent rise of end users has resulted the Anonymous feature to be compromised temporarily. It'll be fixed soon, don't worry

**Comment by camlafit** [comment]
Hello

I think I've same trouble, to be more accurate follow two screenshot.
To note at first run , all display are correct , I can navigate and launch any playlist.
Fileld are emptied once I hide application and reopen it.
I'm connect with a spotify account.

I hope to be on same issue.

Have a nice day

![Capture d’écran du 2024-04-18 11-47-39](https://github.com/KRTirtho/spotube/assets/209915/902a62ea-07e1-496e-af6f-6a338f015e3c)
![Capture d’écran du 2024-04-18 11-47-07](https://github.com/KRTirtho/spotube/assets/209915/6e2bf3c3-601f-4c62-af53-9e63ceead04b)

**Comment by KRTirtho** [comment]
You've to login. The public keys that allowed for Anonymous usage was revoked recently due to high traffic

**Comment by camlafit** [comment]
Hello

In this case is not an anonymous connection, spotube is associate to a free account.
![Capture d’écran du 2024-04-19 09-44-52](https://github.com/KRTirtho/spotube/assets/209915/a53b2b4a-6290-410e-b921-d64932b9c5f8)
![Capture d’écran du 2024-04-19 09-43-48](https://github.com/KRTirtho/spotube/assets/209915/20b06b2b-1ace-4134-ab5a-55dd8550f0d7)

Data disappear only once application is hidden first on notification bar. On start, no trouble. It's possible to explore and do search.
